slices/bash.yaml

```diff
@@ -0,0 +1,10 @@
+package: bash
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libtinfo6_libs
+    contents:
+      # user should manually link /bin/sh to /bin/bash
```

Comment:
you should create this symlink. It deb creates via via a maintainer script, so it should be part of the slice. If not via a mutation script, then as an explicit symlink (using the `symlink` directive)

Reply:
**edit:** apologies. Rafid raised a good point about a potential conflict with `/bin/sh` from the `dash` package. I see you were already doing what I asked in a previous commit and changed it according to Rafid's comment, which is right. It sucks, but I guess the safest option is to do the symlink manually for now. You can therefore ignore this comment.